sources/packages/api/src/services/workflow/workflow.service.ts

```diff
@@ -44,4 +44,21 @@ export class WorkflowService {
       },
     });
   }
+
+  /**
+   * Deletes process instance by a given assessmentWorkflowId.
+   * @param assessmentWorkflowId workflow Id to be deleted.
+   * @returns status code-204 - Request successful (Status 204. No content.).
+   * status code-404 - Process instance with given id does not exist.
+   */
+  async delete(assessmentWorkflowId: string): Promise<number> {
+    const deleteURL = `${this.config.ruleEngineUrl}/process-instance/${assessmentWorkflowId}`;
+    const token = await this.tokensService.simsApiClient.getToken();
+    const response = await axios.delete(deleteURL, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    return response.status;
```

Comment:
I would recommend not returning a number. If I am not wrong, if the status is different from 2xx Axios is going to throw an exception, which means that this method will return always a 2xx.
Can you please consider changing it to return a Promise<void>?